x-pack/plugins/triggers_actions_ui/kibana.jsonc

```diff
@@ -34,7 +34,8 @@
       "esUiShared",
       "kibanaReact",
       "kibanaUtils",
-      "actions"
+      "actions",
+      "cases"
```

Comment:
I added Cases in the required bundles to be able to import some constants. The bundle size of the `triggers_actions_ui` plugin increased by 200 bytes. I think it is acceptable. If you think otherwise let me know so I can hardcode the needed values.

Reply:
I don't think there is an issue with bundles. Only with required plugins. I will verify with operations to be sure. If there is a problem I will duplicate the code.

Reply:
@jcger You were right. CI was complaining about circular dependencies. I will create an issue to create a package so we can import the cases types from there. In this PR I duplicate the code.